plugin-quicklaunch/lxqtquicklaunch.cpp

```diff
@@ -36,13 +36,13 @@
 #include <QFileInfo>
 #include <QLabel>
 #include <QMessageBox>
-#include <QSettings>
 #include <QToolButton>
 #include <QUrl>
-#include <QSettings>
+#include <QDebug>
```

Comment:
...the very same as the previous comment:
https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-quicklaunch/lxqtquicklaunch.cpp#L72, https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-quicklaunch/lxqtquicklaunch.cpp#L77, https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-quicklaunch/lxqtquicklaunch.cpp#L94